libcaf_core/caf/detail/stringification_inspector.test.cpp

```diff
@@ -16,9 +16,9 @@ using namespace std::literals;
 namespace {
 
 struct dummy_user {
-  std::string first_name;
-  std::optional<std::string> last_name;
-  std::optional<std::string> nickname;
+  std::string first_name = {};
+  std::optional<std::string> last_name = {};
+  std::optional<std::string> nickname = {};
```

Comment:
These changes make no sense to me. A struct default-initializes its fields and all fields here have default constructors. When/why would the compiler warn here? We have many structs like this and never initialize fields explicitly in this way.

Reply:
In the test below you do aggregate initialization and only specify some of the fields, leaving the other ones uninitialized. That's where the warning is coming from. There are generally two ways to fix this: Either specify all fields in the aggregate initialization, or specify them as I did here.

Reply:
I also understand that this makes no difference in practice here, but `-Wmissing-field-initializers` is pretty strict about this unfortunately.

Reply:
Got it, thanks. 👍

CI isn't happy about about the changes, though.

Reply:
Should be fixed in a7bd07562113a996902b19f11fb57e867243b342

Reply:
… and 1487b5ef4 for the MSVC build.